### DISCUSSION TITLE
CldImage with Next.js

### DISCUSSION THREAD

**Discussion by richard-chang-coder** [cloudinary-community/next-cloudinary#506] [Q&A]
Curious question

Currently using the CldImage component I am able to crop the image but wonder is there a way to implement coordinates? When I upload the image I am also using the cropping features so when I render the image I was hoping to render the crop version where I cropped. 

I know there is coordinates from the return response just wondering how can I use it. 

Thanks

```
{
    "custom": [
        [
            28,
            7,
            63,
            63
        ]
    ]
}
```

```
<CldImage
				src={'publicId'}
				preserveTransformations
				alt="Description of my image"
				crop={{
					width: 1200,
					height: 1200,

					type: 'thumb',
					source: true,
				}}
				width="500"
				height="500"

			/>
```

**Comment by colbyfayock** [comment]
@richard-chang-coder currently the crop prop doesn't support coordinates, thats something i need to get added and will try to get to this week

Currently the crop option only allows the width/height as you've noticed as well as Zoom: https://github.com/cloudinary-community/cloudinary-util/blob/main/packages/url-loader/src/plugins/cropping.ts

Created an Issue to track this: https://github.com/cloudinary-community/next-cloudinary/issues/507

Just noting context: https://cloudinary.com/documentation/resizing_and_cropping#fixed_coordinates_cropping

**Comment by richard-chang-coder** [comment]
Awesome ,

Thanks 🙏

**Reply by colbyfayock** [reply]
this was released on 6.12.0

**Comment by richard-chang-coder** [comment]
Thanks !
